docs/docs/intro.md

```diff
@@ -0,0 +1,10 @@
+
+# Introduction
+CCom is a language for having the possiblity to define conditional sections in common data formats, which do not allow logic by default. It acts like a pre-processor and can evaluate the conditional expressions, defined by you, based on a JSON data object, which you can pass to CCom along with the input file. <br>
+To get started with CCom, visit our [quick-start guide](../quick-start).
+
+Here you can see a demo, how to use CCom to process a YAML data file with conditional sections:
+
+<script id="asciicast-407484" src="https://asciinema.org/a/407484.js" async></script>
```

Comment:
asciinema looks great 😍

Reply:
Yeah, but I have to replace the Compose Generator clip with the CCom one. Will do that later on ...

Reply:
ah, didn't even notice it 😅

Reply:
is this in your kanban?

Reply:
Yes. Now it is